src/Common/SqlOptions.cs

```diff
@@ -0,0 +1,127 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.ComponentModel;
+using Microsoft.Azure.WebJobs.Hosting;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Sql
+{
+    /// <summary>
+    /// Represents configuration for <see cref="SqlTriggerAttribute"/>.
+    /// </summary>
+    public class SqlOptions : IOptionsFormatter
+    {
+        public const int DefaultBatchSize = 100;
+        public const int DefaultPollingIntervalMs = 1000;
+        private const int DefaultMinimumPollingIntervalMs = 100;
+
+        // NOTE: please ensure the Readme file and other public documentation are also updated if this value ever
```

Comment:
This comment would apply to DefaultBatchSize and DefaultPollingIntervalMs too right?